docs/guides/elements-lookup-troubleshooting.md

```diff
@@ -146,3 +146,13 @@ The final recommendation there would be:
   to improve the corresponding WebDriverAgent sources, but keep in mind there are many automation
   tests around that rely on the current way these attributes are calculated, and we probably don't
   want to break them.
+
+## Symptom #4
```

Comment:
if the element's `value` attribute had 600 bytes words -> if the size of element's `value` attribute is above 512 bytes

Reply:
The desired element is shown in the page tree, but the content of its `value` property is cut off